<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StudentWindow</class>
 <widget class="QWidget" name="StudentWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>سیستم گلستان</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="VL_Program">
     <item>
      <layout class="QHBoxLayout" name="HL_Top">
       <item>
        <widget class="QLabel" name="L_1">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>کاربر :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="L_User">
         <property name="text">
          <string>0000000</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="TB_Signout">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>خروج</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="HS_Top">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTabWidget" name="TW_Program">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="tabsClosable">
        <bool>true</bool>
       </property>
       <property name="movable">
        <bool>false</bool>
       </property>
       <property name="tabBarAutoHide">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="tab_UserMenu">
        <attribute name="title">
         <string>منوی کاربر</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QTabWidget" name="TW_UserMenu">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="tab_Menu_Education">
            <attribute name="title">
             <string>آموزش</string>
            </attribute>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="0" column="0">
              <layout class="QHBoxLayout" name="HL_Education">
               <item>
                <layout class="QVBoxLayout" name="VL_Education">
                 <item>
                  <layout class="QVBoxLayout" name="VL_Education_Buttons">
                   <property name="spacing">
                    <number>0</number>
                   </property>
                   <item>
                    <widget class="QPushButton" name="PB_PresentedCourses">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="text">
                      <string>دروس ارائه شده در ترم</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="PB_WeekSchedule">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="text">
                      <string>برنامه هفتگی دانشجو</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="PB_ExamSchedule">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="text">
                      <string>برنامه امتحان پایان ترم دانشجو</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="PB_Tuition">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="text">
                      <string>مشاهده وضعیت شهریه</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="PB_Assessment">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="text">
                      <string>ارزشیابی اساتید</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="PB_FullInformatin">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="text">
                      <string>اطلاعات جامع دانشجو</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <spacer name="VS_Education">
                   <property name="orientation">
                    <enum>Qt::Vertical</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>20</width>
                     <height>40</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </item>
               <item>
                <spacer name="HS_Education">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="tab_Menu_Enroll">
            <attribute name="title">
             <string>ثبت نام</string>
            </attribute>
            <layout class="QGridLayout" name="gridLayout_4">
             <item row="0" column="0">
              <layout class="QHBoxLayout" name="HL_Enroll">
               <item>
                <layout class="QVBoxLayout" name="VL_Enroll">
                 <item>
                  <layout class="QVBoxLayout" name="VL_Enroll_buttons">
                   <property name="spacing">
                    <number>0</number>
                   </property>
                   <item>
                    <widget class="QPushButton" name="PB_EnrollTime">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="text">
                      <string>مشاهده ساعت ثبت نام</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="PB_PrepEnroll">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="text">
                      <string>ثبت نام مقدماتی</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="PB_MainEnroll">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="text">
                      <string>ثبت نام اصلی</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <spacer name="VS_Enroll">
                   <property name="orientation">
                    <enum>Qt::Vertical</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>20</width>
                     <height>40</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </item>
               <item>
                <spacer name="HS_Enroll">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="tab_Menu_System">
            <attribute name="title">
             <string>سیستم</string>
            </attribute>
            <layout class="QGridLayout" name="gridLayout_5">
             <item row="0" column="0">
              <layout class="QHBoxLayout" name="HL_System">
               <item>
                <layout class="QVBoxLayout" name="VL_System">
                 <item>
                  <widget class="QPushButton" name="PB_ChangePassword">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="text">
                    <string>تغییر گذرواژه</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="VS_System">
                   <property name="orientation">
                    <enum>Qt::Vertical</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>20</width>
                     <height>40</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </item>
               <item>
                <spacer name="HS_System">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
